modules/ROOT/pages/apikit-4-set-header-task.adoc

```diff
@@ -52,7 +52,9 @@ Add the `APIUsageInformation` header declaration:
 
 }
 ----
-<1> This piece of Dataweave code adds headers to the produced output in the Transform Message component.
+[calloutlist]
+.. This piece of DataWeave code adds headers to the produced output in the `Transform Message` component.
```

Comment:
```suggestion
.. This DataWeave code adds headers to the produced output in the `Transform Message` component.
```

Reply:
in or of?

Reply:
For your version, it's "of" :) 
Thanks!